openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -175,95 +175,103 @@ The following subsections illustrate some of the authorization flows supported b
+
+Please note that the diagram does not illustrate all the optional features defined in this specification.
 
 !---
 ~~~ ascii-art
-+--------------+   +-----------+                                    +-------------------+
-| User         |   |   Wallet  |                                    | Credential Issuer |
-+--------------+   +-----------+                                    +-------------------+  
-        |                |                                                    |
-        |    interacts   |                                                    |
-        |--------------->|                                                    |
-        |                |  (1) Obtains Issuer's Credential Issuer metadata   |
-        |                |<-------------------------------------------------->|
-        |                |                                                    |
-        |                |  (2) Authorization Request                         |
-        |                |      (type(s) of Credentials to be issued)         |
-        |                |--------------------------------------------------->|
-        |                |                                                    |
-        |   User Authentication / Consent                                     |
-        |                |                                                    |
-        |                |  (3)   Authorization Response (code)               |
-        |                |<---------------------------------------------------|
-        |                |                                                    |
-        |                |  (4) Token Request (code)                          |
-        |                |--------------------------------------------------->| 
-        |                |      Token Response (access_token)                 |
-        |                |<---------------------------------------------------|    
-        |                |                                                    |
-        |                |  (5) Credential Request (access_token, proof(s))   |
-        |                |--------------------------------------------------->| 
-        |                |      Credential Response                           |
-        |                |      (credential(s) OR transaction_id)           |
-        |                |<---------------------------------------------------|             
++----------+        +-----------+                                    +-------------------+
+| End-User |        |   Wallet  |                                    | Credential Issuer |
++----------+        +-----------+                                    +-------------------+
+    |                     |                                                    |
+    | (1a) End-User       |                                                    |
+    |  selects Credential |  (1b) Credential Offer (credential type)           |
```

Comment:
would it be possible to say, `End-User selects Credential in the Wallet`?

Reply:
I don't mind. But I'd like to be sure we have agreements on the text, as the modification on the sequence diagram isn't much fun

Reply:
let's keep it as-is probably.. @jogu ?

Reply:
I think, given we have the detailed description of step 1a below the diagram, keeping the text in the diagram concise is fine. (i.e. Paul's current text is fine with me.)